.lintrunner.toml

```diff
@@ -338,6 +338,25 @@ init_command = [
 ]
 is_formatter = true
 
+[[linter]]
+code = 'SQLFLUFF'
+include_patterns = ['torchci/clickhouse_queries/**/*.sql']
```

Comment:
Let's start with one file first, i.e. `torchci/clickhouse_queries/workflow_load/query.sql` as I can confirm that one is working correctly.  There is no major issue with the linter adapter, but I think there is another issue with sqlfluff.  It changes the function name from `toUnixTimestamp` into `TOUNIXTIMESTAMP`.   That's a problem on the tool rather than on your adapter.  So let's deal with it in a separate PR.

Reply:
I look into this issue a bit more.  The rule that is responsible for this is CP03 https://docs.sqlfluff.com/en/stable/reference/rules.html#rule-capitalisation.functions. I suggest that we add a `.sqlfluff` file to the repo, then either

* Disable the rule
```
[sqlfluff]
exclude_rules = capitalisation.functions
```

* or make it to use camel-case by default
```
[sqlfluff:rules:capitalisation.functions]
extended_capitalisation_policy = camel
```

Both works, but the second approach looks a bit weird because it changes some one word function like `IF` or `if` into `iF`.